src/ast-helpers.ts

```diff
@@ -11,3 +13,29 @@ export function getReactImportReference(ast): t.ImportDeclaration {
 export function isExportedDeclaration(ast) {
   return t.isExportNamedDeclaration(ast) || t.isExportDefaultDeclaration(ast);
 }
+
+export function findPathInContext(ast, identifierName) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected parameter: 'identifierName' to have a typedef](https://app.codacy.com/gh/wix/vscode-glean/pullRequest?prid=5535229)